Student_BCOs/Derek_Cao_BCO 000534.json

```diff
@@ -0,0 +1,168 @@
+{
+    "bco_id": "http://biocomputeobject.org/BCO_000534",
+    "checksum": "f0634e19eaf3a968002e3f1e40a1f5865ec8f73cca9bf0d05fd594aa17cdabb1",
+    "bco_spec_version": "1.3.0",
+    "provenance_domain": {
+        "name": "Multi-omics profiling reveals key signaling pathways in ovarian cancer controlled by STAT3",
+        "version": "1.0.0",
+        "license": "https://creativecommons.org/licenses/by/4.0/",
+        "created": "2019-10-08T21:21:01.804217",
+        "modified": "2019-10-17T04:34:06.005406",
+        "review": [
+            {
+                "status": "in-review",
+                "reviewer_comment": "Reviewed by BioCompute technical assistant, approval pending modification by author",
+                "reviewer": {
+                    "orcid": "https://orcid.org/0000-0002-8824-4637",
+                    "affiliation": "George Washington University",
+                    "contribution": [
+                        "curatedBy"
+                    ],
+                    "name": "Janisha Patel",
+                    "email": "janishapatel@gwu.edu"
+                },
+                "date": "10/16/2019"
+            }
+        ],
+        "contributors": [
+            {
+                "name": "Derek Cao",
+                "orcid": "https://orcid.org/0000-0002-7901-9470",
+                "affiliation": "George Washington University",
+                "contribution": [
+                    "createdBy"
+                ],
+                "email": "cderek7@gwu.edu"
+            },
+            {
+                "name": "Tiangong Lu",
+                "orcid": "",
+                "affiliation": "Department of Medicinal Chemistry, College of Pharmacy, Rogel Cancer Center, University of Michigan, Ann Arbor",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Armand Bankhead, III,",
+                "orcid": "",
+                "affiliation": "Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor ; Department of Biostatistics, School of Public Health, University of Michigan, Ann Arbor",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Mats Ljungman ",
+                "orcid": "https://orcid.org/0000-0003-1553-6695",
+                "affiliation": "Departments of Radiation Oncology, Rogel Cancer Center, University of Michigan Medical School and Department of Environmental Health Sciences, School of Public Health, University of Michigan, Ann Arbor",
+                "contribution": [
+                    "authoredBy"
```

Comment:
In addition to "authoredBy", each publication author should also have the "derivedFrom" contribution type. Please add. See #112